tensorboard/components/vz_line_chart/vz-line-chart.ts

```diff
@@ -396,11 +417,13 @@ class LineChart {
 
   private linePlot: Plottable.Plots.Line<number|Date>;
   private smoothLinePlot: Plottable.Plots.Line<number|Date>;
+  private marginAreaPlot: Plottable.Plots.Area<number|Date>;
   private scatterPlot: Plottable.Plots.Scatter<number|Date, Number>;
   private nanDisplay: Plottable.Plots.Scatter<number|Date, Number>;
   private yValueAccessor: Plottable.IAccessor<number>;
   private smoothedAccessor: Plottable.IAccessor<number>;
   private lastPointsDataset: Plottable.Dataset;
+  private marginSpecification: MarginSpecification;
```

Comment:
Should the type of this be `MarginSpecification?`? Likewise with other occurrences and with `marginAreaPlot : Plottable.Plots.Area<number|Date>?`, perhaps.

Reply:
SG Done